Terminal.Gui/Views/ComboBox.cs

```diff
@@ -57,14 +57,14 @@ public ComboBox ()
         Initialized += (s, e) => ProcessLayout ();
 
         // On resize
-        SubviewsLaidOut += (sender, a) => ProcessLayout ();
+        SubViewsLaidOut += (sender, a) => ProcessLayout ();
 
-        Added += (s, e) =>
+        IsAddedChanged += (s, e) =>
```

Comment:
Why do we need an IsAdded field? doesn't the SuperView implicitly tell you this.

If you have a SuperView you have been Added to something otherwise you have not.

Reply:
Good question. `IsAdded` has been there for a while.

@BDisp do you remember?

Reply:
Yes I remember. It was added since many years on the v1. We all know that if a `SuperView` is null then the view wasn't added yet but there was a necessity of raise some code in the exact moment a view is added to a SuperView by raising an event. Without raising this event a view needed to check periodically, using a timer or timeout, if SuperView is not null.

Reply:
But I understand that the `IsAdded` property can be removed, since it's easy to check that. Only the virtual method and event is important.